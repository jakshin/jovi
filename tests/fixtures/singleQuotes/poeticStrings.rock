(Possessives don't work in poetic string literals... they become poetic number literals,
always starting with the digit 4 because "says" has 4 characters)
The world's says Hello
Say the world

Your mom is hot
Somebody says hey to your mom's sexy sister
Say Somebody
