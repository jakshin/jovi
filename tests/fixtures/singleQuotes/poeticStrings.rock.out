45
hey to your mom's sexy sister
